app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -14,12 +14,13 @@ def submit
       # Once we run this job asynchronosuly, this data translation can be moved into the
       # async `perform` method
       form_content = JSON.parse(request.body.string)
+      uploads = form_content['form526'].delete('attachments')
       converted_form_content = EVSS::DisabilityCompensationForm::DataTranslation.new(
         @current_user, form_content
       ).translate
       response = service.submit_form(converted_form_content)
       EVSS::IntentToFile::ResponseStrategy.delete("#{@current_user.uuid}:compensation")
-      EVSS::DisabilityCompensationForm::SubmitUploads.start(@current_user, response.claim_id)
+      EVSS::DisabilityCompensationForm::SubmitUploads.start(@current_user, response.claim_id, uploads)
```

Comment:
Could you make calling this conditional on whether there's uploads or not? That way a job won't spawn if there's nothing for it to do :)